core/fragment.c

```diff
@@ -1711,6 +1711,7 @@ fragment_exit()
                                   false /* no flush */);
     DELETE_LOCK(client_flush_request_lock);
 #endif
+    return;
```

Comment:
If you didn't talk offline about the label I would think this was superfluous.  Look at the review patch view of this: it looks useless.  This needs a comment to avoid someone deleting it.